sql/src/main/java/io/crate/analyze/expressions/ExpressionAnalyzer.java

```diff
@@ -267,12 +268,21 @@ protected Symbol convertFunctionCall(FunctionCall node, ExpressionAnalysisContex
     }
 
     @Nullable
-    private WindowDefinition getWindowDefinition(Optional<Window> maybeWindow, ExpressionAnalysisContext context) {
+    private WindowDefinition getWindowDefinition(Optional<Window> maybeWindow,
+                                                 ExpressionAnalysisContext context) {
         if (!maybeWindow.isPresent()) {
             return null;
         }
+        var overWindow = maybeWindow.get();
```

Comment:
Maybe rename to something like:

```suggestion
        var unresolvedWindow = maybeWindow.get();
        final Window resolvedWindow;
```

To clarify the difference between `overWindow` and `window`